tests/unit/components/ilios-calendar-event-test.js

```diff
@@ -0,0 +1,17 @@
+import { moduleForComponent, test } from 'ember-qunit';
```

Comment:
This one tests the CP directly. I could remove this and check if the icon shows or not in the integration test (I can only that indirectly in integration tests).